src/contrib/cluster/Akka.Cluster.Metrics.Tests.MultiNode/ClusterMetricsExtensionSpec.cs

```diff
@@ -17,6 +17,7 @@
 using Akka.Configuration;
 using FluentAssertions.Extensions;
 using ConfigurationFactory = Akka.Configuration.ConfigurationFactory;
+using MultiNodeFactAttribute = Akka.MultiNode.TestAdapter.MultiNodeFactAttribute; 
```

Comment:
Need to add this using on all of our spec file because we now have 2 `MultiNodeFactAttribute` class, one in the old implementation and one in the new standalone implementation.